storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3451,6 +3454,8 @@ static void rocksdb_update_table_stats(
 
     io_perf_read.bytes = table_handler->m_io_perf_read.bytes.load();
     io_perf_read.requests = table_handler->m_io_perf_read.requests.load();
+    lock_wait_timeout_stats = table_handler->m_lock_wait_timeout_counter.load();
+    table_handler->m_lock_wait_timeout_counter.clear();
```

Comment:
The only concern is whether this will work well if run concurrently. It seems like there's a chance that we double count the number of lock wait timeouts?